docs: add keep screen alive entry (#4116)
mealie-recipes/mealie: docs/docs/documentation/getting-started/faq.md

- **Developer Experience:** Without a database, a lot of the work to maintain your data is taken on by the developer instead of a battle-tested platform for storing data.
- **Multi User Support:** With a solid database as backend storage for your data, Mealie can better support multi-user sites and avoid read/write access errors when multiple actions are taken at the same time.

> ## Why is there no Keep Screen Alive button when I access a Mealie Recipe?

Reviewer: ```suggestion
## Why is there no "Keep Screen Alive" button when I access a recipe?
```

> You've perhaps visited the Mealie Demo and noticed that it had a Keep Screen Alive button, but it doesn't show up in your own Mealie instance.

Reviewer: ```suggestion
You've perhaps visited the Mealie Demo and noticed that it had a "Keep Screen Alive" button, but it doesn't show up in your own Mealie instance.
```

> There are typically two possible reasons for this.

Reviewer: ```suggestion
There are typically two possible reasons for this:
```

> - You're accessing your Mealie instance without using HTTPS. The Wake Lock API is only available if HTTPS is used. Read more here: https://developer.mozilla.org/en-US/docs/Web/API/Screen_Wake_Lock_API

Reviewer: ```suggestion
1. You're accessing your Mealie instance without using HTTPS. The Wake Lock API is only available if HTTPS is used. Read more here: https://developer.mozilla.org/en-US/docs/Web/API/Screen_Wake_Lock_API
```

> - Or, you're accessing your Mealie instance on a browser which is not supported. You can test this here: https://vueuse.org/core/useWakeLock/#demo

Reviewer: ```suggestion
2. You're accessing your Mealie instance on a browser which doesn't support the API. You can test this here: https://vueuse.org/core/useWakeLock/#demo
```
